_posts/2018-01-25-review-things-3.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Review: Things 3 for Mac and iOS"
+subtitle: A task manager you will actually want to use
+custom_type: post
+review:
+  rating: 4.5
+  type: app
+  app:
+    developer: Cultured Code
+  price: 9.99-$49.99
+  vendor: Mac App Store
+  url: https://itunes.apple.com/us/app/things-3/id904280696?mt=12
+---
+
+{% figure extendout /uploads/2018/01/things-review-01.jpg "Things 3 on iPad." %}
+
+I've never been a fan of todo apps. I just don't use them. People everywhere seem to swear by these things. There's much talk about the whole GTD mentality, and what app was the best for getting things done.
+
+{% include toc.html %}
+
+I came across Things years ago. I owned the first version of the app, but didn't end up using it all that much. Recently someone I know started using Things 3 and I was intrigued. I'd been using Trello to keep track of my ideas for articles on this site, video ideas, and jobs that I'd applied to. I have a love/hate relationship with Trello. I feel that parts of the app are very well designed, and others not so much.
+
+After watching and reading some positive reviews, I decided to give Things 3 a try. Cultured Code offers a fifteen-day trial of the Mac app.
+
+I was hooked. Things 3 is the first todo app that I actually use. I attribute it in part to the design, but also to where I am in life. These days, I feel I forget things a lot more and need to write down things I need to get done.
```

Comment:
> These days, I feel I forget things a lot more and need to write down things I need to get done.

Something strange going on here. It's a mouthful of words that I stumbled over a few times to get down. Maybe something more like this?

These days I am a much more forgetful of the things I need to get done and find I need to write them down.